README.md

```diff
@@ -26,13 +26,16 @@ ember install ember-frost-object-browser
 
 ### Action Bar
 
-| Attribute        | Type          | Value | Description                                         |
-| ---------------- | ------------- | ----- | --------------------------------------------------- |
-| controls         | `array`       |       | Controls that will be available in the action bar   |
-| isLoading        | `boolean`     | false | **default** - Action bar is not in a loading state  |
-|                  |               | true  | Action bar is in a loading state                    |
-| loadingText      | `string`      |       | Text that appears beside the loading animation      |
-| selectedItems    | `array`       |       | List of items that are currently selected           |
+| Attribute             | Type          | Value | Description                                                 |
+| --------------------- | ------------- | ----- | ----------------------------------------------------------- |
+| controls              | `array`       |       | Controls that will be available in the action bar           |
+| isLoading             | `boolean`     | false | **default** - Action bar is not in a loading state          |
+|                       |               | true  | Action bar is in a loading state                            |
+| isVisible             | `boolean`     |       | defaults to computed prop based on selectedItems emptiness  |
+| loadingText           | `string`      |       | Text that appears beside the loading animation              |
+| moreActionButtons     | `array`       |       | Controls that will be wrapped up into 'More...' button      |
```

Comment:
struggling with `moreActionButtons`  as it is a `button` with `more actions`

would it be more appropriately named `moreActionsButton`?  and `moreActionsButtonText`?

Reply:
yeah - the naming is funny. I was following what happened in forch. But, perhaps it would make more sense to call the property just 'moreActions'.

Reply:
yes i prefer `moreActions` and `moreActionsText`